test-integration/DepositZap.js

```diff
@@ -0,0 +1,347 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+const { bytes32 } = require("../utils/helpers");
+const timeMachine = require("ganache-time-traveler");
```

Comment:
Thoughts on upgrading hardhat and using those new network helpers for new tests we write?